### DISCUSSION TITLE
Markdown Cell Editing Bar && Code Cell "Up" and "Down" Arrows

### DISCUSSION THREAD

**Discussion by AndrewLaganaro** [microsoft/vscode-jupyter#10162] [New Ideas]
### **Markdown Cell Editing Bar**

Just like we have on github itself or in google colab, it'd be great if we have available **Markdown buttons** to trigger its functions quickly and visually

**Colab:**
![image](https://user-images.githubusercontent.com/45211616/170771016-cff44bd2-84a7-40eb-8f3e-05b15741904e.png) 
**Github:**
![image](https://user-images.githubusercontent.com/45211616/170771405-105bce84-d882-4c38-b708-867eda1f6e2e.png)

### **Code Cell "Up" and "Down" Arrows**

Just like we have on google colab, it'd be great if we have available Up and down buttons to **move the cell** by clicking on them.
Today we only have the "execute cell above" or "execute cell below" options.

**Colab**
![image](https://user-images.githubusercontent.com/45211616/170771463-6dd6423d-75ff-44d2-8055-62cf7cf6a1aa.png)

**Current Vscode options:**
![image](https://user-images.githubusercontent.com/45211616/170771466-81fe7301-d82e-49c6-b607-eb247162bb21.png)

**Comment by rchiodo** [comment]
Thanks for the suggestion. It could actually be added by any extension as VS code core handles markdown. Adding new buttons is just a matter of adding the commands and buttons to an extension.
